redis/commands/cluster.py

```diff
@@ -634,6 +633,14 @@ def cluster_shards(self, target_nodes=None):
         """
         return self.execute_command("CLUSTER SHARDS", target_nodes=target_nodes)
 
+    def cluster_myshardid(self, target_nodes=None):
+        """
```

Comment:
Looks like you copied CLUSTER SHARDS comment, can you update to the appropriate comment